lib/Runtime/Base/FunctionBody.cpp

```diff
@@ -4349,6 +4349,14 @@ namespace Js
         this->RecordConstant(location, str);
     }
 
+    void FunctionBody::RecordBigIntConstant(RegSlot location, LPCOLESTR psz, uint32 cch, bool isNegative)
+    {
+        ScriptContext *scriptContext = this->GetScriptContext();
+        JavascriptBigInt *result = RecyclerNew(scriptContext->GetRecycler(), JavascriptBigInt, scriptContext->GetLibrary()->GetBigIntTypeStatic());
+        Var bigintConst = JavascriptBigInt::ToVar(psz, cch, isNegative, scriptContext, result);
```

Comment:
ToVar sounds like a plain cast, not an initialization with side effects. I'd prefer to pass all of the psz, cch, isNegative to the constructor (via RecyclerNew in the previous line) so we don't need a second step here. If we need a second method for initialization for some reason, I'd prefer a name like Initialize.